src/lib/notcurses.c

```diff
@@ -594,6 +594,16 @@ inline int ncplane_cursor_move_yx(ncplane* n, int y, int x){
   return 0;
 }
 
+inline int ncplane_cursor_move_rel(ncplane* n, int y, int x){
```

Comment:
i don't think there's much point in putting `inline` on an `API` function, because callers can't "see" beyond the shared object boundary. but i'll get that,no worries.

Reply:
That never occurred to me, but thanks for another learning opportunity.